include/rocksdb/advanced_options.h

```diff
@@ -156,6 +156,26 @@ enum class PrepopulateBlobCache : uint8_t {
   kFlushOnly = 0x1,  // Prepopulate blobs during flush only
 };
 
+// Bitmask enum for verify output options during compaction.
+// This allows fine-grained control over what verification is performed
+// on compaction output files and when it's enabled.
+enum VerifyOutputOptions : uint32_t {
```

Comment:
Avoid enum (without class) unless it's already scoped in something limited already. rocksdb:: is a big scope.

See https://softwareengineering.stackexchange.com/a/204566 for how to avoid excessive casting.